docs/release/trg-8/trg-8-00.md

```diff
@@ -0,0 +1,93 @@
+---
+title: TRG 8.00 - Security Scanning Toolchain
+---
+
+| Status | Created     | Post-History                         |
+|--------|-------------|--------------------------------------|
+| Active | 26-Feb-2024 | Initial release                      |
+
+## Why
+
+Our primary aim is to improve security and define best practices across the Tractus-X ecosystem.
+
+## Description
+
+In this document we will provide instructions regarding security tools and GitHub Security tab.
+
+### Emphasized Guidelines for Optimizing Security GitHub Actions
+
+:::info
+
+These developer-friendly workflows ensure all security findings are directly visible in the **GitHub Security tab**, keeping everything within the familiar GitHub environment. While using these actions is optional, their implementation is strongly recommended as they introduce a foundational level of security into the development process.
+
+:::
+
+Following, here are specific guidelines and best practices for developers:
+
+1.**Action Failures:** Actions should only fail if there is an error with the Action "Engine" itself or if there is a misconfiguration in the workflow. Failures should not occur based solely on high-severity findings.
+
+2.**Manual Execution:** Include the on: workflow_dispatch option in all workflows. This allows you to manually trigger workflows whenever necessary.
+
+3.**Scheduling Workflows:** Workflows should run frequently. Ideally, they should be set to execute once nightly. At a minimum, they should run once per week. Configure the on: schedule option to achieve this frequency.
+
+4.**Exclusions:** Do not exclude files or directories from scans. If false positives are detected, they can be simply ignored. However, when pushing documentation to the main branch, the workflows do not need to be executed. For such cases, configure the exclude option.
+
+5.**Pull Requests (PRs):** It's not mandatory for workflows to run with every PR. Nonetheless, the Secret Scan is strongly recommended and deemed sufficient.
+
+6.**Target Scanning:** Avoid over-scanning. Focusing on scanning the releases and the main branch from which releases are made is adequate.
+
+7.**Issue Reporting:** Should developers encounter issues during scanning or have questions regarding tool usage, they are encouraged to create an issue in our repository. An appropriate issue template has been provided to streamline this process.
+
+By adhering to these guidelines, developers can efficiently integrate GitHub actions into their workflow, ensuring optimal security without compromising productivity.
+
+:::caution
+
+To pass the quality gates, all **critical**, **high** and **medium** security vulnerabilities **must be mitigated**.
+This generic statement may be in conflict with TRG 4.02 for container scans as TRG 4.02 states that base images should be used as-is..

```

Comment:
There are two dots at the end of the sentence. Also you removed the sentence the subsequent sentence (The statement from TRG 4.02 **prevails**.), could you please add the sentence again?

Reply:
@evegufy  I've just changed it, thank you for pointing this out

Reply:
@klaudiaZF sure, there's still a space missing "as-is.The".

Reply:
@evegufy I fixed that, thank you